### ISSUE TITLE
_config.yml not configured yet

### ISSUE THREAD

**Issue by garyng** [Relib/relib.github.io#1]
Merely import, everything is not configured.

**Comment by zypeh** [comment]
Do you think that we should consider others implementation ? like `hexo` I mean

**Comment by garyng** [comment]
@zypeh yes, but reorganizing the posts layout and updating content should be prioritized.
